src/css/jwplayer/imports/logo.less

```diff
@@ -6,13 +6,15 @@
     cursor: pointer;
     pointer-events: all;
 
-    background-repeat: no-repeat;
+    background: no-repeat right;
     background-size: contain;
 
     top: auto;
     right: auto;
     left: auto;
     bottom: auto;
+    max-height: 15%;
```

Comment:
Do we want this to be 15% of the player height? For example, in your test config, the eggplant logo is a 300x300 square, so instead of the `jw-logo` being 54x54, it is 96x54.